src/CHANGES.txt

```diff
@@ -17,6 +17,10 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
       the index from find() was not used.
     - Turn previously deprecated debug options into failures:
       --debug=tree, --debug=dtree, --debug=stree, --debug=nomemoizer.
+    - If SCONS_CACHE_MSVC_CONFIG env var is set, scons will cache the
```

Comment:
change from "env var" to "shell environment variable" so we don't confuse with env =Environment().. ?

Reply:
Also write something like use with care. It's not guaranteed to invalidate contents if you change the MSVC version you're request nor if you update/install new versions of MSVC on the system.

Reply:
py2.7 builds have tipped over: `NameError: global name 'FileNotFoundError' is not defined`. I can fix, obviously; getting tired of keeping track of 2v3 diffs!!!

Reply:
looks like IOError for py27?

Reply:
> Also write something like use with care. It's not guaranteed to invalidate contents if you change the MSVC version you're request nor if you update/install new versions of MSVC on the system.

In case of a big change, it will make a separate cache entry - the pathname to the batch file is part of the dictionary key, so if I switch from 2019 to 2017, that will create/use a different entry.  For finer-grained version selection (well, we can't actually *select* at the moment, but there could be an upgrade that bumps version), this is not detected - as noted in the submit message above,  We'll need to decide if that should be a blocker.